build_tools/waf_dynamo.py

```diff
@@ -626,7 +626,7 @@ def default_flags(self):
         self.env.append_value('DEFINES', ['DM_TEST_DLIB_HTTPCLIENT_NO_HOST_SERVER'])
 
         for f in ['CFLAGS', 'CXXFLAGS']:
-            self.env.append_value(f, ['-Wall', '-fPIC', '-fno-exceptions', '-fno-rtti',
+            self.env.append_value(f, ['-Wall', '-fPIC', '-fno-exceptions', '-fno-rtti', '-Wno-nontrivial-memcall',
```

Comment:
we've started getting annoying compile warnings from Emscripten about `memset(this, ...)`

Reply:
Maybe it's worth to create a separate ticket and accumulate all kinds of warning which compiler reports and then fix it?

Reply:
In this case, I've already checked in this flag, as it was overly spammy.
Fixing it makes little sense as it's merely convertiing `memset(ptr, ..)` to `memset((void*)ptr, ...)`. It doesn't really fix anything.